scipy-1.0/paper.tex

```diff
@@ -974,16 +974,17 @@ \subsubsection*{Benchmark suite}
 The airspeed velocity (asv\cite{asvref}) library enables benchmarking Python packages
 over their lifetimes, and the performance of the SciPy code base was monitored
 with asv starting in February of 2015 (PR \#4501). In addition to ensuring that
-unit tests are passing (see above), confirming that performance generally
+unit tests are passing (see below), confirming that performance generally
```

Comment:
Figure \ref{fig:asvbench}

Reply:
This I don't get -- the "see below" refers to the Test Suite subsubsection below, not the figure.

Reply:
My mistake, but this is all the more reason to reference that section than to leave it ambiguous. I think it would help to give the section a label and reference it just as you would a figure.

Reply:
If you'd prefer I can merge as-is and add this when I take a pass.